requirements/base.txt

```diff
@@ -15,6 +15,7 @@ xlrd==0.9.3
 django-moj-irat>=0.4
 # https://github.com/celery/kombu/issues/1063
 kombu==4.6.3
+dj-database-url==0.5.0
```

Comment:
_nitpick_ If I see this dependency in a group of "locked because problems" dependencies (between kombu and itertools), I would assume it has problems as well. What do you think about moving it to the "main" group?